src/main/java/com/feywild/feywild/events/LootModifierEvent.java

```diff
@@ -0,0 +1,26 @@
+package com.feywild.feywild.events;
+
+import com.feywild.feywild.FeywildMod;
+import com.feywild.feywild.events.loot_modifiers.MineshaftChestLootModifier;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.common.loot.GlobalLootModifierSerializer;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+import javax.annotation.Nonnull;
+
+@Mod.EventBusSubscriber(modid = "feywild", bus = Mod.EventBusSubscriber.Bus.MOD)
```

Comment:
No other code uses `@EventBusSubscriber`. Also you can register `GlobalLootModifierSerializer`s through the LibX registration system. Just add `@RegisterClass` and put the `GlobalLootModifierSerializer` into a `public static final` field.

Reply:
Like this? 
`    @RegisterClass
    public static final class Serializer extends GlobalLootModifierSerializer<MineshaftChestLootModifier> `

Cause that gives me an error

Reply:
Could you assist pls?

Reply:
Like this:

```java
@RegisterClass
public class LootModifiers {

    public static final GlobalLootModifierSerializer<?> mineshaftChestAdditions = new MineshaftChestLootModifier.Serializer();
}